lib/ldclient-rb/ldclient.rb

```diff
@@ -182,6 +182,31 @@ def track(event_name, user, data)
       add_event({:kind => 'custom', :key => event_name, :user => user, :data => data })
     end
 
+    #
+    # Returns the key of every feature
+    #
+    def feature_keys
+      get_features.map {|feature| feature[:key]}
+    end
+
+    #
+    # Returns all features
+    #
+    def get_features
+      res = @client.get (@config.base_uri + '/api/features') do |req|
```

Comment:
Ah, makes sense. It sounds like we'll probably want both-- `/api/features` is what you want for your use case, and we can add a separate resource at `/api/eval/features` for the evaluable collection of flags.